### ISSUE TITLE
Shortcut plugin does not work on globally installed thelounge

### ISSUE THREAD

**Issue by igalic** [MiniDigger/thelounge-plugin-shortcuts#16]
When installing thelounge globally:

```sh
# npm install --global --unsafe-perm thelounge
```

and then installing this plugin as:

```sh
$ env THELOUNGE_HOME=/srv/thelounge thelounge install thelounge-plugin-shortcuts
```

 this plugin fails to load with the following error:

```
[ERROR] Package thelounge-plugin-shortcuts could not be loaded: Error: Cannot find module 'thelounge/src/models/msg'
[DEBUG] Error: Cannot find module 'thelounge/src/models/msg'
     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
     at Function.Module._load (internal/modules/cjs/loader.js:562:25)
     at Module.require (internal/modules/cjs/loader.js:692:17)
     at require (internal/modules/cjs/helpers.js:25:18)
     at Object.<anonymous> (/srv/thelounge/packages/node_modules/thelounge-plugin-shortcuts/index.js:3:13)
     at Module._compile (internal/modules/cjs/loader.js:778:30)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
     at Module.load (internal/modules/cjs/loader.js:653:32)
     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

as pointed out in the IRC channel, this could be related to the fact that this plugin requires a newer version of the msg plugin than the one released in 3.3.0 (the version i have installed)

**Comment by GewoonYorick** [comment]
Have you tried running it on a newer version? (Source or the new 4.0 pre release)

Then we can check if the older thelounge version is indeed the issue.

**Comment by MiniDigger** [comment]
fixed in v1.0.12
